modules/cnf-rfhe-notifications-api-refererence.adoc

```diff
@@ -0,0 +1,276 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+[id="cnf-rfhe-notifications-api-refererence_{context}"]
+= Redfish hardware event notifications REST API reference
+
+You can use the Redfish hardware fast event notifications REST API to subscribe an application to the Redfish hardware events for the parent node.
+
+Subscribe DU applications to Redfish hardware events using the resource address `/cluster/node/<node_name>/redfish/event`, where `<node_name>` is the cluster node running the DU application.
+
+Status request are sent by making a REST API `PUT` call to `/subscriptions/status/<subscription_id>`. The API call returns events through the AMQP event bus to the subscribed address.
```

Comment:
this paragraph is not needed as status API is described below in detail. It also sounds like the status request returns events which is not ture.